manifests/master/puppetdb_conf.pp

```diff
@@ -30,9 +31,16 @@
       value   => $port,
     }
   } else {
-    ini_setting { 'puppetdbserver_urls':
-      setting => 'server_urls',
-      value   => "https://${server}:${port}/",
+    unless $servers {
+      ini_setting { 'puppetdbserver_urls':
+        setting => 'server_urls',
+        value   => "https://${server}:${port}/",
+      }
+    } else {
+      ini_setting {'puppetdbserver_urls':
```

Comment:
Could we put a warning here if `$server and $port` are set to something other than the default, i.e. someone tried to set both urls and those settings?